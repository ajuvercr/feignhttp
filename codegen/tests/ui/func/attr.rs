use feignhttp_codegen::get;

// error: unknown arg type: aaa
//    |  pub async fn send_get(#[aaa] a: i32) -> Result<String> {}
//    |                          ^^^

#[get("http://xxx")]
pub async fn send_get(#[aaa] a: i32) -> Result<String> {}

fn main() {}
