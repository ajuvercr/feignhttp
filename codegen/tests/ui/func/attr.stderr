error: unknown arg type: aaa
 --> tests/ui/func/attr.rs:8:25
  |
8 | pub async fn send_get(#[aaa] a: i32) -> Result<String> {}
  |                         ^^^
